cmd/mattermost/commands/plugin.go

```diff
@@ -167,9 +167,12 @@ func pluginListCmdF(command *cobra.Command, args []string) error {
 	}
 	defer a.Shutdown()
 
-	if pluginsResp, err := a.GetPlugins(); err != nil {
-		CommandPrintErrorln("Unable to list plugins. Error: " + err.Error())
-	} else {
+	pluginsResp, appErr := a.GetPlugins()
+	if appErr != nil {
+		return errors.New("Unable to list plugins. Error: " + appErr.Error())
+	}
+
+	if pluginsResp != nil {
```

Comment:
GetPlugins never will return you pliginsResp as nil if the error is nil. Said that, if you want to be more "defensive" here, I would use something like:
```go
if pluginsResp == nil {
    return nil
}
```
This way we maintain the happy path in the lowest level of indentation.

Reply:
Or maybe:
```go
if pluginsResp == nil {
    return errors.New("Unable to list plugins. Invalid plugins Response.")
}
```

Reply:
Sounds good. I can replace appError with the blank identifier (_)

Reply:
We are using it, previously, but pluginsResp wont be nil, unless the plugins implementation changes.